packages/component-library/draft/Kaizen/Modal/Modal.elm

```diff
@@ -297,6 +298,19 @@ viewContent (Config config) =
                             Nothing ->
                                 confirmationConfig
 
+                    withOnConfirmDisabled confirmationConfig =
+                        case configs.onConfirm of
```

Comment:
Hmm... Do we need to couple the onConfirm with the disabled state? In my mind the button can be disabled without there being an onConfirm msg. Also if I change the order of withOnConfirm below to be after `onConfirmDisabled` then this will always result in `False`.

Reply:
Thankssss! This looks way better now 🙏 :D